hack/apply-terraform

```diff
@@ -43,10 +44,14 @@ function main () {
     -w /env/terraform
     ${TERRAFORM}
   )
+
+  # Clear out any cached local module state.
+  rm -rf ${env_dir}/terraform/.terraform/modules
+
   ${terraform[@]} init
   # Hard-coding kubeconfig path for now because there's a ticket to
   # stop writing it to local disk at all.
-  ${terraform[@]} apply -var="kubeconfig_dir_path=/env/.kubeconfig"
```

Comment:
It looks like you removed the hard-coded kubeconfig path, so maybe remove the comment above as well